third_party/tizen/tizen_qemu.py

```diff
@@ -153,7 +159,7 @@ def whereis(binary_name):
 status = 0
 # Run QEMU.
 with open(args.output, "wb") as output:
-    logging.info("run: %s", " ".join(map(shlex.quote, qemu_args)))
+    log.info("run: %s", " ".join(map(shlex.quote, qemu_args)))
```

Comment:
I think that by default the info level is not shown. However, this "run: ..." log message is really helpful when debugging issues (so one can run qemu manually). Please make sure that it is visible.